tests/unit/Develop.spec.js

```
@@ -0,0 +1,29 @@
+import { mount } from '@vue/test-utils'
+import { nextTick } from 'vue'
+import Develop from '@/components/Develop.vue'
+
+describe('Develop.vue', () => {
```

ESLint

[error] 'describe' is not defined.

(no-undef)